api/assessments/routes.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from api.responses import assessment_201_response
+from api.responses import error_response
+from db.models.assessment import AssessmentError
+from db.models.assessment import AssessmentMethods
+from flask import request
+from flask import Response
+from flask.views import MethodView
+
+
+class AssessmentsView(AssessmentMethods, MethodView):
+    def search(self):
+        """
+        GET /assessments endpoint
+        :return: Json Response
+        """
+        return Response(
+            json.dumps(self.assessments(as_json=True)),
+            mimetype="application/json",
+        )
+
+    def get(self, assessment_id: str):
+        """
+        GET /assessments/{assessment_id} endpoint
+        If the assessment_id matches a valid assessment in the db
+        return the assessment
+        If no matching valid link is found returns a 404 error message
+        :param assessment_id: id of the assessment
+        :return: 200 assessment JSON / 404 Error
+        """
+        try:
+            assessment = self.get_by_id(assessment_id)
+        except AssessmentError as e:
+            return error_response(404, e.message)
```

Comment:
❤️ ❤️ Love the custom errors, will dovetail nicely with the new logging we will have soon!

Reply:
👌